packages/slate-tools/tools/webpack/config/prod.js

```diff
@@ -6,6 +6,7 @@ const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
 const SlateConfig = require('@shopify/slate-config');
 const SlateTagPlugin = require('@shopify/slate-tag-webpack-plugin');
+const SlateLocalesPlugin = require('slate-locales-plugin');
```

Comment:
You'll want to add this to `core.js` since we want to do the transformation in both dev and prod builds.